public/wp-content/themes/ColoredCow/all_soiree.php

```diff
@@ -12,35 +12,33 @@
 		// query_posts($args);
 		if($loop->have_posts()):
 			while ($loop->have_posts()) : $loop->the_post();
-			$last_date = strtotime(get_field('last_date'));
-			$current_date = strtotime(date("d/m/y"));
+			$last_date = get_field('last_date');
+			$current_date = date("Y-m-d");
 	?>			
 				<div class="container">
 					<div class="row">
-						<div class="col-md-12">	
-							<h1 class="page-title text-center"><a href="<?php if($current_date <= $last_date){echo get_home_url();}?>"><?php echo get_the_title();?></a></h1>
+						<div class="col-xs-12">	
+							<h1 class="page-title text-center"><?php if($current_date <= $last_date):?><a href="<?php echo get_home_url();?>"
```

Comment:
breaking this line into multiple might help to make the code clean and non-confusing.